app/assets/stylesheets/styles/_presets.css.scss

```diff
@@ -285,7 +285,7 @@ textarea {
     @include font-awesome-mixin();
 
     cursor: pointer;
-
+    color: #b7ca98;
```

Comment:
Color literals like `#b7ca98` should only be used in variable declarations; they should be referred to via variable everywhere else.